lib/cli/autocomplete.js

```diff
@@ -94,3 +91,11 @@ utils.iterativeCompletion = (inputs, depth, completer) => {
   }
   return [completions, help];
 };
+
+module.exports = {
+  filterKeys,
+  split,
```

Comment:
I changed `rsplit` to `common.rsection` because it is not compatible with `String.prototype.split()` This `split` is also not compatible with `String.prototype.split()` so I think we will either rename it or use `String.prototype.split()` instead.